oga/frontend/src/containers/QuestionList/NewQuestion/NewQuestion.js

```diff
@@ -95,8 +95,9 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = dispatch => {
   return {
-    createQuestion: (type, content, target_location) =>
-      dispatch(actionCreators.createQuestion({ author: "HI", type: type, content: content, target_location: target_location})),
+    createQuestion: (type, target_location) =>
```

Comment:
Again, are we creating answers or questions?

Reply:
I think this page is for making question, as it's name is NewQuestion.js